tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph/graph_component.css

```diff
@@ -28,10 +80,69 @@ limitations under the License.
 }
 
 .op-container {
-  border: 1px solid #c6cad1;
-  border-radius: 3px;
-  display: table;
-  height: max-content;
-  margin: 0 auto;
-  width: 300px;
+  border: 2px solid #c6cad1;
+  border-radius: 4px;
+  box-shadow: 1px 3px #eee;
+  cursor: pointer;
```

Comment:
is this element interactable? If so, use proper semantic tags like `button` (also proper tab-index="-1").